test/unit/bootstrap.coffee

```diff
@@ -1,21 +1,25 @@
 window.ngLeafLetTestGlobals = {}
 
+afterEach ->
+  window.doTestLog = false
```

Comment:
Lighten up the logging. Only turn doTestLog explicitly if you need to debug logs in your spec.